angular_code_standards.md

```diff
@@ -0,0 +1,107 @@
+# Angular Development Standards & Conventions
+
+This document outlines recommended coding standards and practices for Angular projects. Following these guidelines promotes consistency, readability, and maintainability across the codebase.
+
+## 1\. Project Structure & File Organization
+
+We advocate for a **feature-centric architecture**, where application sections are encapsulated in their own folders.
+
+- **Feature Modules:** Each major application feature should reside in its dedicated folder and possess its own Angular feature module.
+- **Routing Segregation:** Keep feature-specific routes contained within their respective modules, minimizing cross-feature routing.
+- **Lazy Loading Focus:** Design the structure with lazy-loading in mind to optimize initial bundle size.
+- **Module Contents:** Each feature module typically includes:
+
+  - A primary **feature module file** (e.g., feature.module.ts).
+  - A **feature-level routing file** for managing internal routes (e.g., feature-routing.module.ts).
+  - A pages or views directory for **container components** (components loaded directly by routes).
+  - A components or presentational directory for **reusable UI components** specific to the module.
+  - Dedicated folders for module-specific services, models, enums, directives, and pipes (as needed).
+  - An optional state folder if using NgRx or similar state management.
+
+- **Single-Purpose Files:** Ensure each file is dedicated to a single concern (e.g., one component per file, one service per file). This extends to enums, which should also reside in their own files.
```

Comment:
It might be helpful to include a brief explanation of what container components are.